web/src/pages/admin/providers/setup/[id]/index.tsx

```diff
@@ -114,6 +118,10 @@ const Page = () => {
     if (data != null) {
       await mutate({ ...data, status: "VALIDATING" }, { revalidate: false });
       const res = await validateProvidersetup(id);
+
+      if (res?.configValidation.length > 0) {
```

Comment:
I think this might be better as a useEffect
It should reset the state if the length is 0

Reply:
Disregard this! my mistake